__tests__/app.test.js

```diff
@@ -15,3 +23,33 @@ describe('GET /api', () => {
       });
   });
 });
+
+describe('GET /api/topics', () => {
+  test('200: Responds with an array of topic objects containing slug and description properties', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+});
+
+describe('Error handling: non-existent route queried', () => {
```

Comment:
I like that you've put this in a separate describe block - applies to all non-existing endpoints